Added basic login and Sign up forms

cheddartot opened the pull request:
basic, unformated HTML pages added with sign up and login forms, values passed to routes.py for use in API later

cheddartot committed:
Added basic login and Sign up forms

basic, unformated HTML pages added with sign up and login forms, values passed to routes.py for use in API later

pgayee694 commented on .idea/inspectionProfiles/profiles_settings.xml:
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
stuff in ide specific folders (like the .idea folder) should not be commited

pgayee694 commented on app/forms.py:
```diff
@@ -0,0 +1,14 @@
+from flask_wtf import FlaskForm
```
add flask_wtf to requirements.txt

pgayee694 commented on app/routes.py:
```diff
@@ -1,5 +1,23 @@
+            form_log_in.username.data, form_log_in.remember_me.data))
+        return redirect('/')
+    return render_template('login.html', title='Log In', form=form_log_in)
+@app.route('/signup/', methods=['GET', 'POST'])
```
have a blank line between functions

pgayee694 commented on app/templates/login.html:
```diff
@@ -0,0 +1,23 @@
+    <!DOCTYPE html>
```
fix indentation in this file

pgayee694 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,22 @@
+ <!DOCTYPE html>
```
fix indentation in this file

pgayee694 commented on app/forms.py:
```diff
@@ -0,0 +1,14 @@
+
+class SignInForm(FlaskForm):
+    username = StringField('UsernameSignUp', validators=[DataRequired()])
+    password = PasswordField('PasswordSignUp', validators=[DataRequired()])
```
These field names show up really weird on the the web-page. These could probably be changed to 'Username' and 'Password' respectively

pgayee694 reviewed (changes requested):
Looks good, just a few changes

cheddartot committed:
Fixed commited files and weird sign up text

added the .idea folder to the gitignore, and flask_wtf stuff to the requirements.txt file. Also changed the names of a few variables so the sign in page does't look so bad.

cheddartot committed:
html indentation fix

Fixed html indentation and added a few missing closing tags

tkorisko reviewed (approved):
All looks good, but add location form on signup.

cheddartot committed:
Added location fields

added fields for location and 32 char limit warning

cheddartot committed:
added password confirmation

Added a second field to confirm the entered password, and moved 32 char limit warning to top of page

cheddartot committed:
Added size validators to sign in

added validators to make sure fields were under 32 characters to be accepted by Sign in
